config/config.json

```diff
@@ -12,7 +12,8 @@
         "version": "1.0.0",
         "artifactVerificationPolicies": {
             "application/vnd.cncf.notary.v2": "any",
-            "org.example.sbom.v0": "all"
+            "org.example.sbom.v0": "all",
+            "application/vnd.ratify.spdx.v0": "any"
```

Comment:
I believe spdx has an iana type already  - https://www.iana.org/assignments/media-types/text/spdx - `text/spdx`

Reply:
Updated config and guide to reflect `text/spdx` as the type.